packages/client/src/views/RecentActivity.vue

```diff
@@ -1,7 +1,15 @@
 <template>
   <section class="container">
     <b-card class="border-0">
+    <div class="d-flex">
       <h4 class="card-title gutter-title1 row">Recent Activity</h4>
+      <div class="justify-content-end left-margin">
+        <b-button @click="exportCSV" :disabled="loading" variant="outline-secondary">
+        <b-icon icon="download" class="mr-1 mb-1" font-scale="0.9" aria-hidden="true" />
+          Export to CSV
+          </b-button>
```

Comment:
Nit: the closing `</b-button>` tag is indented further than the opening `<b-button>` tag on line 7.

Reply:
You're totally right - missed that!